src/System.Private.CoreLib/src/Internal/Runtime/Augments/RuntimeThread.Windows.cs

```diff
@@ -274,7 +274,7 @@ private bool CreateThread(GCHandle thisThreadHandle)
         }
 
         /// <summary>
-        /// This an entry point for managed threads created by applicatoin
+        /// This an entry point for managed threads created by application
```

Comment:
Since you're touching this line - would you mind also fixing the missed "is" in "This an entry point"?

The same comment is also duplicated in RuntimeThread.Unix.cs (you could fix that for extra credit :)).